CONTRIBUTING.md

```diff
@@ -89,7 +89,7 @@ It’s possible that you found a bug or enhancement that we haven’t posted to
     ```
 9. [Open a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork), and make sure to fill in the required fields.
 10. An Expensify engineer and a member from the Contributor-Plus team will be assigned to your pull request automatically to review.
-11. Provide daily updates until reaching completion of your PR.
+11. Daily updates on weekdays are expected. If there is a plan for no updates for a while we expect the courtesy to be informed about the duration there will be no work so that we may plan accordingly. Any issue without an update for 1 week with no explanation why there is no progress will be considered abandoned and will likely be closed or reassigned, and the original contract will be terminated.
```

Comment:
Just pulling in that feedback from Alex B since I liked the phrasing and adding a few things for tone...

>Daily updates on weekdays are highly recommended.

>If you know you won’t be able to provide updates for > 1 week, please comment on the PR or issue how long you plan to be out so that we may plan accordingly. We understand everyone needs a little vacation here and there.

This could maybe be shortened 

>Any issue that doesn't receive an update after 1 week may be considered abandoned and the original contract terminated.

Reply:
I think this phrasing is very clear:
> Daily updates on weekdays are highly recommended. Any issue that doesn't receive an update after 1 week may be considered abandoned and the original contract terminated.

Reply:
Love the shortened point 👍 I think it could be nice to touch up the phrase "Any issue that doesn't receive an update **after 1 week** may be considered abandoned..." - Something like this:

> Any issue that doesn't receive an update **for 1 full week** may be considered abandoned...

Reply:
Will you be updating the copy here @mallenexpensify or would you like one of us to update it?

Reply:
Can one of you update the copy (if only cuz I'm not sure of the fastest/best way to do it :ohnothing:)